docker/netutils/Pipfile

```diff
@@ -8,6 +8,7 @@ verify_ssl = true
 [packages]
 requests = {extras = ["socks"],version = "*"}
```

Comment:
can we add websockets/ntlm/dnspython/aiohttp ? 
Lets make sure we're adding here everything we can